File: python-projects/fizzbuzz.py
start = int(input("Enter your start value: "))
stop = int(input("Enter your stop value: "))

for x in range(start, stop):
    if x % 3 == 0 and x % 5 == 0:
        print("FizzBuzz", end=" ")
    elif x % 3 == 0:
        print("Fizz", end=" ")
    elif x % 5 == 0:
        print("Buzz", end=" ")
    else:
        print(x, end=" ")

File: dart-projects/fizzbuzz.dart
void main() {
  var startNum;
  // var endNum = 50;
  for (startNum = 1; startNum <= 50; startNum++){
    if (startNum % 3 == 0 && startNum % 5 == 0){
      print("FizzBuzz");
    }
    else if (startNum % 3 == 0){
      print('fizz ');
    }
    else if (startNum % 5 == 0){
      print("buzz ");
    }
    else{
      print(startNum);
    }
  }

}